components/ui/tabcounter/src/main/java/mozilla/components/ui/tabcounter/TabCounterMenu.kt

```diff
@@ -0,0 +1,54 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.ui.tabcounter
+
+import android.content.Context
+import androidx.core.content.ContextCompat.getColor
+import mozilla.components.browser.menu2.BrowserMenuController
+import mozilla.components.concept.menu.MenuController
+import mozilla.components.concept.menu.candidate.DrawableMenuIcon
+import mozilla.components.concept.menu.candidate.TextMenuCandidate
+import mozilla.components.concept.menu.candidate.TextStyle
+
+open class TabCounterMenu(
+    context: Context,
+    private val onItemTapped: (Item) -> Unit
+) {
+    open class Item {
+        object CloseTab : Item()
+        object NewTab : Item()
```

Comment:
Fenix version uses the Tab mode here. Should we deal with custom vs private vs normal modes in AC?

Reply:
https://github.com/mozilla-mobile/fenix/blob/bd2c4079430673b69500f73b0519b3e3433db590/app/src/main/java/org/mozilla/fenix/components/toolbar/TabCounterMenu.kt#L30